backend/core/test/listings/listings.e2e-spec.ts

```diff
@@ -280,12 +280,12 @@ describe("Listings", () => {
     const listings = res.body.items
 
     // The Coliseum seed has the soonest applicationDueDate (1 day in the future)
-    expect(listings[0].name).toBe("Test: Coliseum")
+    expect(listings[0].name).toContain("Test: Coliseum")
```

Comment:
Can you test for the exact match instead with the doorway prefix, since we know the full name?

Reply:
yes!